brownie/project/main.py

```diff
@@ -740,8 +746,16 @@ def load(project_path: Union[Path, str, None] = None, name: Optional[str] = None
         if not name[0].isalpha():
             raise BadProjectName("Project must start with an alphabetic character")
         name = "".join(i for i in name.title() if i.isalnum())
-    if next((True for i in _loaded_projects if i._name == name), False):
-        raise ProjectAlreadyLoaded("There is already a project loaded with this name")
+
+    already_loaded_project: Optional[Project] = None
+    for loaded_project in _loaded_projects:
+        if loaded_project._name == name:
+            already_loaded_project = loaded_project
+
+    if already_loaded_project is not None:
+        if raise_if_loaded:
+            raise ProjectAlreadyLoaded("There is already a project loaded with this name")
+        return already_loaded_project
```

Comment:
I _think_ you can shorten this a bit by nesting the `if` statement on 755 inside the first loop.

```
for loaded_project in _loaded_projects:
    if loaded_project._name == name:
        if raise_if_loaded:
            raise ProjectAlreadyLoaded("There is already a project loaded with this name")
        else:
            return loaded_project
```
Note: I haven't actually checked out the patch and ran the tests though.

Reply:
@jummy123 : You are correct, my friend. Fixed.